controllers/kubeadmbootstrapconfig_controller.go

```diff
@@ -65,7 +68,31 @@ func (r *KubeadmBootstrapConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Res
 	}
 
 	// Get the machine
+	machine := &v1alpha2.Machine{}
+	machineKey := client.ObjectKey{
+		Namespace: req.Namespace,
+		Name:      machineRef.Name,
+	}
+
+	if err := r.Get(ctx, machineKey, machine); err != nil {
```

Comment:
I will update the cluster-api dependency and use the util @vincepri pointed out that does this in a followup PR

Reply:
This just merged https://github.com/kubernetes-sigs/cluster-api/pull/1159